docs/source/endpoints/inherit.md

```diff
@@ -0,0 +1,55 @@
+---
+myst:
+  html_meta:
+    "description": "The inherit endpoint shows values inherited from content items higher in the hierarchy."
+    "property=og:description": "The inherit endpoint shows values inherited from content items higher in the hierarchy."
+    "property=og:title": "Inherit behaviors"
+    "keywords": "Plone, plone.restapi, REST, API, inherit, acquisition, behaviors"
+---
+
+(inherit-behaviors-label)=
+
+# Inherit behaviors
+
+The `@inherit` service makes it possible to inherit data from behaviors defined on ancestors in the object hierarchy.
+
+```{tip}
+Inheriting behaviors is similar to the concept of {term}`acquisition` in Zope, but it doesn't happen automatically, so it's safer.
+```
+
+To use the service, send a `GET` request to the `@inherit` endpoint in the context of the content item that is the starting point for inheriting.
+Specify the `expand.inherit.behaviors` parameter as a comma-separated list of behaviors.
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/inherit_get.req
+```
+
+For each behavior, the service will find the closest ancestor which provides that behavior.
+The result includes `from` (the `@id` and `title` of the item from which values were inherited)
+and `data` (values for any fields that are part of the behavior).
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/inherit_get.resp
+:language: http
+```
+
+Ancestor items for which the current user lacks the View permission will be skipped.
+
+(inherit-behaviors-expansion-label)=
+
+## Expansion
+
+This endpoint can be used with the {doc}`../usage/expansion` mechanism which allows getting more information about a content item in one query, avoiding unnecessary requests.
+
+You can make a `GET` request for a content item, and include parameters to request `inherit` expansion for specific behaviors:
```

Comment:
```suggestion
You can make a `GET` request for a content item, and include parameters to request `inherit` expansion for specific behaviors.
```

Reply:
Not worth the effort. Documentation follows https://6.docs.plone.org/contributing/documentation/authors.html#vale-for-american-english-spelling-grammar-and-syntax-and-style-guide, specifically https://learn.microsoft.com/en-us/style-guide/punctuation/colons. Maybe next time.

Reply:
@stevepiercy I can't find a specific reference to this in the Microsoft style guide, but Microsoft's Azure REST API docs also use a colon to introduce request/response examples, for example in https://learn.microsoft.com/en-us/rest/api/azure/#send-the-request

Reply:
Sometimes docs don't follow their own style guide, like that _never_ happens in Plone. 😏